File: mobile-app/memeory/lib/components/common/channel_logo.dart
import 'package:cached_network_image/cached_network_image.dart';
import 'package:flutter/material.dart';
import 'package:memeory/util/env.dart';

class ChannelLogo extends StatelessWidget {
  const ChannelLogo({Key key, this.channelId}) : super(key: key);

  final String channelId;

  @override
  Widget build(BuildContext context) {
    return Container(
      width: 35,
      height: 35,
      decoration: BoxDecoration(
        borderRadius: BorderRadius.circular(10),
        image: DecorationImage(
          image: CachedNetworkImageProvider(
            '${env.backendUrl}/channels/logo/$channelId',
          ),
        ),
      ),
    );
  }
}

File: web-app/src/app/common/logo/ChannelLogo.js
import React from 'react';
import {getChannelLogoUrl} from "../../../api/channels";

export const ChannelLogo = ({channelId, width = 30, height = 30, ...props}) => {

    return (
        <img src={getChannelLogoUrl(channelId)}
             width={width}
             height={height}
             style={{
                 margin: "0 auto",
                 width: width,
                 height: height,
                 borderRadius: 5
             }}
             {...props}
             alt={channelId}
        />
    );
};